src/Distribution/Server/Framework/Instances.hs

```diff
@@ -250,7 +250,9 @@ instance SafeCopy CompilerFlavor where
     putCopy JHC               = contain $ putWord8 7
     putCopy LHC               = contain $ putWord8 8
     putCopy UHC               = contain $ putWord8 9
+#if !MIN_VERSION_Cabal_syntax(3,16,0)
```

Comment:
Wouldn't it make sense to drop sorry for HaskellSuite as well?

Reply:
> There is no mention of HaskellSuite removal in https://github.com/haskell/cabal/blob/master/release-notes/Cabal-3.16.0.0.md. Is that a mistake in code or changelog?

I see: 

> Remove knowledge about hmake and haskell-suite https://github.com/haskell/cabal/pull/10912

Reply:
@ulysses4ever that's a breaking change in library API. It's significant change.

Reply:
Fair point, thank you! Will try to adjust the process so that we get all API changes marked as significant.

Reply:
Also, removing `HaskellSuite` constructor was completely unnecessary, AFACIT.

There is e.g. `Helium` and being in this (pseudo)enum is the only reference of it.

Reply:
> Also, removing `HaskellSuite` constructor was completely unnecessary, AFAICT.

Yeah, maybe.

> There is e.g. `Helium` and being in this (pseudo)enum is the only reference of it.

At least `Helium` once existed, in contrast to `HaskellSuite`: https://hackage.haskell.org/package/helium